config.json

```diff
@@ -904,7 +916,13 @@
         "name": "Grep",
         "uuid": "78bcbae1-a0f2-460c-ba89-e51844fe9397",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "strings",
+          "for-loops",
+          "array-analysis",
+          "template-strings"
+        ],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/grep/solutions/TomPradat